common/src/main/kotlin/streams/utils/JSONUtils.kt

```diff
@@ -236,7 +236,7 @@ abstract class StreamsTransactionEventDeserializer<EVENT, PAYLOAD: Payload> : Js
         ?.properties
         ?.mapValues {
             if (points.contains(it.key)) {
-                org.neo4j.values.storable.PointValue.fromMap((it.value as Map<String, Any>).toMapValue())
+                org.neo4j.values.storable.PointValue.fromMap((it.value as Map<String, Any>).toMapValue().filter { _, u -> u != Values.NO_VALUE })
```

Comment:
you could instead update `streams.utils.JSONUtilsKt#toMapValue` to something like this:

```kotlin
fun Map<String, Any?>.toMapValue(): MapValue {
    return this
        .filterValues { v -> v != null }
        .entries
        .fold(MapValueBuilder()) { builder, entry -> 
            builder.add(entry.key, Values.of(entry.value, false)); builder}
        .build()
}
```

I even wonder if we need an extension function like this at all... `toMapValue` is only used in 2 tests and here. A regular function (or no function at all) would maybe work as well. 
Please lmk what you think :)

Reply:
yes, this also crossed my mind but I think this is a general method that can also be used in other places to map values. This is why I decided to add the NO_VALUE filter in the point specific method. But of course getting rid of toMapValue method could also work.

Reply:
maybe this then?

```kotlin
fun Map<String, Any?>.toMapValue(allowNulls: boolean): MapValue {
    return this
        .filterValues { v -> allowNulls || v != null }
        .entries
        .fold(MapValueBuilder()) { builder, entry -> 
            builder.add(entry.key, Values.of(entry.value, allowNulls)); builder}
        .build()
}
```

Reply:
yes, I think this works